### PR TITLE
chore: Update instructions with up to date names

### LINKED ISSUE #84: Update docs to latest terminology (mostly s/@/at/
### Is your feature request related to a problem? Please describe.

The instructions were last updated a couple of years ago, so they're still littered with `@sign` rather than `atSign` and `secondary` rather than `atServer` etc.

### Describe the solution you'd like

Replace `@` with `at` where appropriate

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Areadrill**
closes #84 

**- What I did**
replaced `@sign` with `atSign` and `secondary` with `atServer` and adjusted grammar where needed.

**- How I did it**
N/A

**- How to verify it**
N/A

**- Description for the changelog**
Update Instructions with more up to date names

### CODE REVIEW

File: Instructions.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ your atSign DNS and port number
 
 ## If you want to start over
 
-To completely delete the data from a atServer so that a new set of keys will
+To completely delete the data from ab atServer so that a new set of keys will
```
Comment (cpswan) [inline Instructions.md]: Typo: `s /ab/an/`

**Review by cpswan — CHANGES_REQUESTED** [review]
Thanks @Areadrill it's a good start.

Sadly I can't approve and merge yet as there's a typo.

Also the line:

> `Fire up an @ app like @buzz or @wavi`

Need attention e.g. Fire up an atApp like atBuzz or atWavi

**Comment by buzz** [comment]
Hey there, could you please abide from tagging my username (buzz) in this PR discussion as I don't want to receive notifications for it? Thank you very much!

File: Instructions.md (hunk 1 above)
Comment (Areadrill) [reply Instructions.md]: ah, gotcha. Sorry for the oversight

**Review by cpswan — APPROVED** [review]

**Pull request merged**